app/views/movies/_movie_rating.html.erb

```diff
@@ -5,7 +5,7 @@
         <icon class="fa fa-heart"></icon>
         <span class="pseudo-header"> Your Enjoyment:</span> <%= movie.ratings.by_user(current_user).first.value %>/10
         <%= link_to '<i class="fa fa-pencil"></i> Edit'.html_safe,
-                    movie_rating_path(movie, movie.ratings.by_user(current_user).first),
+                    edit_movie_rating_path(movie, movie.ratings.by_user(current_user).first),
```

Comment:
I switched this to the `edit` path so the user doesn't have to stop by an annoying show page for a rating before getting to the edit functionality. This makes sense to me because the link to get to this rating page said "Edit".